cropper/app/controllers/CropperController.scala

```diff
@@ -52,23 +50,44 @@ class CropperController(auth: Authentication, crops: Crops, store: CropStore, no
 
   def export = auth.async(parse.json) { httpRequest =>
     httpRequest.body.validate[ExportRequest] map { exportRequest =>
+      implicit val logMarker: LogMarker = MarkerMap(
+        "requestType" -> "export",
+        "requestId" -> RequestLoggingFilter.getRequestId(httpRequest)
+      )
+
       val user = httpRequest.user
       val onBehalfOfPrincipal = auth.getOnBehalfOfPrincipal(user)
 
       executeRequest(exportRequest, user, onBehalfOfPrincipal).map { case (imageId, export) =>
+
         val cropJson = Json.toJson(export).as[JsObject]
         val updateMessage = UpdateMessage(subject = UpdateImageExports, id = Some(imageId), crops = Some(Seq(export)))
         notifications.publish(updateMessage)
 
         Ok(cropJson).as(ArgoMediaType)
 
       } recover {
-        case InvalidSource => respondError(BadRequest, "invalid-source", InvalidSource.getMessage)
-        case ImageNotFound => respondError(BadRequest, "image-not-found", ImageNotFound.getMessage)
-        case InvalidImage => respondError(BadRequest, "invalid-image", InvalidImage.getMessage)
-        case MissingSecureSourceUrl => respondError(BadRequest, "no-source-image", MissingSecureSourceUrl.getMessage)
-        case InvalidCropRequest => respondError(BadRequest, "invalid-crop", InvalidCropRequest.getMessage)
-        case ApiRequestFailed => respondError(BadGateway, "api-failed", ApiRequestFailed.getMessage)
+        case InvalidSource =>
+          logger.error(logMarker, InvalidSource.getMessage)
+          respondError(BadRequest, "invalid-source", InvalidSource.getMessage)
+        case ImageNotFound =>
+          logger.error(logMarker, ImageNotFound.getMessage)
+          respondError(BadRequest, "image-not-found", ImageNotFound.getMessage)
+        case InvalidImage =>
+          logger.error(logMarker, InvalidImage.getMessage)
+          respondError(BadRequest, "invalid-image", InvalidImage.getMessage)
+        case MissingSecureSourceUrl =>
+          logger.error(logMarker, MissingSecureSourceUrl.getMessage)
+          respondError(BadRequest, "no-source-image", MissingSecureSourceUrl.getMessage)
+        case InvalidCropRequest =>
+          logger.error(logMarker, InvalidCropRequest.getMessage)
+          respondError(BadRequest, "invalid-crop", InvalidCropRequest.getMessage)
+        case ApiRequestFailed =>
+          logger.error(logMarker, ApiRequestFailed.getMessage)
+          respondError(BadGateway, "api-failed", ApiRequestFailed.getMessage)
+        case NonFatal(e) =>
+          logger.error(logMarker, s"export failed with exception", e)
+          respondError(InternalServerError, "unknown-error", e.getMessage)
```

Comment:
is there a reason we've not moved the logger line into `respondError`

Reply:
`respondError` is shared between all of the services (common-lib, part of the Argo helpers), and I wasn't prepared to audit that logging from it worked for every service in this PR (eg. do all have an implicit logMarkers in scope? are we duplicating other logging? etc.)

One day we'll get all of the services behaving in a common way...